### ISSUE TITLE
click_on "identificator", visible: false doesn't work

### ISSUE THREAD

**Issue by asiniy** [keathley/wallaby#111]
subj.

@keathley I found that 

``` elixir
find("identificator", visible: false) # => works well
click_on("identificator", visible: false) # => doesn't work, it writes me element was found but its not visible to a real user
```

**Comment by jpettit** [comment]
Is there any sort of workaround for this issue?

**Comment by keathley** [comment]
@jpettit you can find the element and then chain that with `click`. Something like this:

```elixir
find("identifier", visible: false)
|> click()
```

This will be fixed with the fix for #114. It should be in the next release.

**Comment by keathley** [comment]
This has been fixed with the new `Query` module.
